.github/workflows/release.yml

```diff
@@ -58,6 +58,14 @@ jobs:
       python_version: ${{ needs.pyversion.outputs.pyversion }}
       poetry_version: 1.4.2
 
+  unit_tests:
+    name: Validate all Unit Tests pass
+    needs: [pyversion]
+    uses: ./.github/workflows/sub_unittest.yml
+    with:
+      python_version: ${{ needs.pyversion.outputs.pyversion }}
+      poetry_version: 1.4.2
```

Comment:
@alperenkose  you already know about this, but we need to get rid of the `poetry_version` completely

Reply:
done.